webapp/src/ts/modules/contacts/contacts-edit.component.ts

```diff
@@ -5,7 +5,8 @@ import { isEqual as _isEqual } from 'lodash-es';
 import { ActivatedRoute, Router } from '@angular/router';
 
 import { LineageModelGeneratorService } from '@mm-services/lineage-model-generator.service';
-import { EnketoFormContext, EnketoService } from '@mm-services/enketo.service';
+import { EnketoService } from '@mm-services/enketo.service';
+import { EnketoFormContext } from '@mm-services/enketo-form.service';
```

Comment:
Would it be worth renaming this at the same time?

```suggestion
import { EnketoFormService } from '@mm-services/enketo-form.service';
```

Reply:
TLDR: I don't think we want to rename `EnketoFormContext` (unless I am totally missing @garethbowen point...)

@m5r, I think you are correct here.  Previously, the `enketo.service` file exported both `EnketoFormContext` and `EnketoService`.  The `EnketoService` was the actual Angular service to use for interacting with Enekto. `EnketoFormContext` is just an interface that defined the parameters of some of the public functions on the `EnketoService`. It was exported to allow consumers of the `EnketoService` to build properly typed arguments to pass in.  

After the refactor, I moved `EnketoFormContext` into the `enketo-form.service` file (since it was also being used there internally).